### PR TITLE
1739-accepting-referrals

### PR DESCRIPTION
**Pull request by jazzdragon**

### CODE REVIEW

**Comment by jazzdragon** [comment]
To test, create a referral record in your db with the referral form, then sign up with `?invitation_id=...`

File: src/app/auth/sign-up/sign-up.component.ts [hunk 1]
```diff
@@ -38,10 +39,9 @@ export class SignUpComponent implements OnInit {
     private feedback: UserFeedbackService,
     private userSessionService: UserSessionService,
   ) {
-    if (this.route.snapshot.queryParams.email) {
-      // convert spaces to + because of url encoding
-      this.email = this.route.snapshot.queryParams.email.replace(/ /g, '+')
-    }
+    // convert spaces to + because of url encoding
+    this.email = this.route.snapshot.queryParams.email?.replace(/ /g, '+')
+    this.invitationId = this.route.snapshot.queryParams.invitation_id
```
Comment (ShiftyMcCool) [inline src/app/auth/sign-up/sign-up.component.ts:44]: Is this being used anywhere?

**Review by ShiftyMcCool — APPROVED** [review]
Just a small question and 👍

**Comment by ShiftyMcCool** [comment]
I noticed your Slack message about needing to create SF entries for referrals on sign-up so the fact that this needs a SF user to complete is fine

**Pull request merged**